lib/prism/ripper_compat.rb

```diff
@@ -406,6 +469,23 @@ def visit_binary_op_assign(node, operator: node.operator)
       on_opassign(on_var_field(ident_val), op_val, visit(node.value))
     end
 
+    # In Prism this is a CallNode with :[] as the operator.
+    # In Ripper it's an :aref.
+    def visit_aref_node(node)
+      first_arg_val = visit(node.arguments.arguments[0])
```

Comment:
I don't think we can make the assumption that there's only one argument, but I'm fine delaying this if you want to handle it later. Same for aref_field.

Reply:
Oh, huh, Ruby will take up to two. But yeah, there's a *lot* here that I haven't done with full generality. Still adding things like... Hash tables at all :-)